package.json

```diff
@@ -86,7 +85,6 @@
     "sequelize-cli": "^6.4.1",
     "swagger-ui-react": "^4.15.5",
     "tiny-emitter": "^2.1.0",
-    "use-debounce": "^9.0.4",
```

Comment:
Another package already depends on use-debounce, we don't need to write our own, and can keep the dependency. plus; use-debounce is a way more complete version with multiple checks and balances